lms/djangoapps/completion/models.py

```diff
@@ -217,3 +217,19 @@ def __unicode__(self):
             block_key=self.block_key,
             completion=self.completion,
         )
+
+
+class UserSalesforceContactId(models.Model):
```

Comment:
this model doesn't have any relationship with completion , so you should move this class to another django app for example `student`

Reply:
Yes, i didn't know where it should written. @andrey-canon